### PR TITLE
Integer & Fibonacci

### PR DESCRIPTION
**Pull request by sylviama**
Although the branch name is call "Integer" but actually it's both integer and Fibonacci. Code and unit tests

### CODE REVIEW

**Comment by VikingPaul** [comment]
All I can see that you could change is you have commented out code that you don't need.

**Comment by VikingPaul** [comment]
:+1:

**Comment by bernardanderson** [comment]
:+1:

File: Mathmagician.Tests/FibnocciTests.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        }
+
+        [TestMethod]
+        public void FibTestLength()
+        {
+            Fibonacci fib = new Fibonacci();
+            int[] answer3 = fib.PrintFib(12);
+            Assert.AreEqual(12, answer3.Length);
+        }
+
```
Comment (katerebekah) [inline Mathmagician.Tests/FibnocciTests.cs:51]: Try a test using CollectionAssert.AreEqual to compare the list you expect with the list your class returns.

File: Mathmagician.Tests/IntegerTests.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        }
+
+        [TestMethod]
+        public void testIntLength()
+        {
+            Integer int4 = new Integer();
+            int[] answer4 = int4.PrintInteger(40);
+            Assert.AreEqual(40, answer4.Length);
+        }
+
```
Comment (katerebekah) [inline Mathmagician.Tests/IntegerTests.cs:58]: Make a test with CollectionAssert.AreEqual here, too.

**Comment by sylviama** [comment]
@katerebekah

**Comment by katerebekah** [comment]
:+1:

**Pull request merged**